src/main/java/seedu/address/model/event/Event.java

```diff
@@ -0,0 +1,26 @@
+package seedu.address.model.event;
+
+public class Event {
+    //TODO: [Event] Complete the class
+    private final String eventName;
+
```

Comment:
Each event will still need to have a list of students i.e. List<Person> to keep track which students are attending which event.
Will need an accessor to the list and a way to add students to the list also.

Reply:
Okay, I have added a field to manage the list of attendees tot the `Event` class.

I didn't add the way to add students to the list yet - this is still a dummy implementation and there might be some design considerations you can make for associating persons to events (e.g. unmodifiable lists, etc).